site/content/docs/5.2/customize/color.md

```diff
@@ -6,15 +6,373 @@ group: customize
+        <div class="px-3 py-2 rounded-2" style="background-color: var(--bs-light-border-subtle); color: var(--bs-light-text);">Border subtle</div>
+      </td>
+      <td>
+        {{< markdown >}}`--bs-light-border-subtle`{{< /markdown >}}
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <div class="px-3 py-2 h-100 rounded-2" style="background-color: var(--bs-light-text); color: var(--bs-body-bg);">Text</div>
+      </td>
+      <td>
+        {{< markdown >}}`--bs-light-text`{{< /markdown >}}
+      </td>
+    </tr>
+    <tr>
+      <td rowspan="4">
+        {{< markdown >}}**Dark —** Additional theme option for higher contrasting colors.{{< /markdown >}}
+      </td>
+      <td class="ps-0">
+        <div class="p-3 rounded-2 text-bg-dark border">Dark</div>
+      </td>
+      <td>
+        {{< markdown >}}`--bs-dark`<br>`--bs-dark-rgb`{{< /markdown >}}
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <div class="px-3 py-2 rounded-2 border" style="background-color: var(--bs-dark-bg-subtle); --bs-border-color: var(--bs-dark-border-subtle); color: var(--bs-dark-text);">Background subtle</div>
+      </td>
+      <td>
+        {{< markdown >}}`--bs-dark-bg-subtle`{{< /markdown >}}
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <div class="px-3 py-2 rounded-2" style="background-color: var(--bs-dark-border-subtle); color: var(--bs-dark-text);">Border subtle</div>
+      </td>
+      <td>
+        {{< markdown >}}`--bs-dark-border-subtle`{{< /markdown >}}
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <div class="px-3 py-2 h-100 rounded-2" style="background-color: var(--bs-dark-text); color: var(--bs-body-bg);">Text</div>
+      </td>
+      <td>
+        {{< markdown >}}`--bs-dark-text`{{< /markdown >}}
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Using the new colors
+
+These new colors are accessible via CSS variables and utility classes—like `--bs-primary-bg-subtle` and `.bg-primary-subtle`—allowing you to compose your own CSS rules with the variables, or to quickly apply styles via classes. The utilities are built with the color's associated CSS variables, and since we customize those CSS variables for dark mode, they are also adaptive to color mode by default.
+
+{{< example >}}
+<div class="p-3 text-primary-emphasis bg-primary-subtle border border-primary-subtle rounded-3">
+  Example element with utilities
+</div>
+{{< /example >}}
+
+### Theme colors
 
 We use a subset of all colors to create a smaller color palette for generating color schemes, also available as Sass variables and a Sass map in Bootstrap's `scss/_variables.scss` file.
 
 <div class="row">
   {{< theme-colors.inline >}}
   {{- range (index $.Site.Data "theme-colors") }}
     <div class="col-md-4">
-      <div class="p-3 mb-3 bg-{{ .name }} {{ if .contrast_color }}text-{{ .contrast_color }}{{ else }}text-white{{ end }}">{{ .name | title }}</div>
+      <div class="p-3 mb-3 text-bg-{{ .name }} rounded-3">{{ .name | title }}</div>
```

Comment:
`.bg-light` on light mode is not white so `.bg-dark` shouldn't be black on dark mode shouldn't it? Maybe more dark gray.

Reply:
So the intent with these new Sass/CSS variables and their utilities is that these new ones are responsive to color mode changes. The existing colors are not, I don't think they accurately can be honestly. Not yet at least—that'd be a bit of a mess to be changing for all the mixed use cases we've used these colors for (in text color, border color, bg color). We need a big more nuance, which the new colors aim to add.